pyproject.toml

```diff
@@ -1,5 +1,119 @@
 [build-system]
-requires = ["setuptools>=45", "setuptools_scm[toml]>=6.2"]
+requires = ["setuptools>=63", "setuptools_scm[toml]>=7"]
 build-backend = "setuptools.build_meta"
 
+[project]
+# https://peps.python.org/pep-0621/#readme
+requires-python = ">=3.7"
```

Comment:
I wonder is there any decent `pyproject.toml` formatter?

Reply:
yep, prettier, i will configure it after this gets in.

Reply:
https://github.com/tox-dev/pyproject-fmt